src/managers/data_manager.py

```
@@ -747,51 +747,123 @@ def _build_flat_data_structure(self, raw_data: List[Dict]) -> bool:
+                logger.debug(f"Data was flattened: {len(flattened_data)} nodes extracted")
+
+                # 平坦化されたデータを処理
+                id_key = "id"  # 平坦化後のIDキー
+                children_key = "children"  # 平坦化後の子要素キー
+
+                # 平坦化されたデータからdata_mapとchildren_mapを構築
+                all_child_ids = set()
+
+                for item in flattened_data:
+                    if isinstance(item, dict):
+                        # IDを取得（既存のIDまたはパスベースのID）
```

Ruff (0.14.6)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)